workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -88,32 +86,27 @@ public async Task Connect(string workerType, ILogDispatcher logger)
                 }
 
                 var origin = transform.position;
-                ConnectionDelegate connectionDelegate;
                 var chosenService = GetConnectionService();
                 var connectionParameters = GetConnectionParameters(workerType, chosenService);
+                SpatialOSConnectionHandlerBuilder connectionHandlerBuilder = new SpatialOSConnectionHandlerBuilder();
```

Comment:
now that we have the `ConnectionHandlerBuilder`, it would be really nice to get rid of the `WorkerConnector`. If we can make it a `WorkerBuilder` as Jamie suggested, I think we can get rid of this and simply provide sample implementations of how to use it for PC vs Mobile. 
This would allow us to finally get rid of the many levels of inheritance we have right now

Reply:
I wanted to keep Unity and non Unity separate. The worker connector is glue between the worker and Unity, so I suspect that some form of it will still be useful.